event_routing_backends/utils/xapi_lrs_client.py

```diff
@@ -71,8 +72,15 @@ def bulk_send(self, statement_data):
             requests.Response object
         """
         logger.debug('Sending {} xAPI statements to {}'.format(len(statement_data), self.URL))
+        response = None
 
-        response = self.lrs_client.save_statements(statement_data)
+        try:
+            response = self.lrs_client.save_statements(statement_data)
+        except JSONDecodeError:
+            logger.warning(f"Events already in LRS: {statement_data}")
```

Comment:
This error can be triggered for any JSON issues. Since we have no ability to tell the difference between a real error and a spurious 204, maybe the message can be `"JSON Decode Error, this may indicate that all sent events are already stored."`?